core/init.c

```diff
@@ -36,6 +36,14 @@
 #include <auto_init.h>
 #endif
 
+#ifndef CONFIG_BOOT_MSG
+#define CONFIG_BOOT_MSG 1
+#endif
+
+#if !defined(CONFIG_BOOT_MSG_STRING)
```

Comment:
Just a styling suggestion, feel free to ignore.

```suggestion
#ifndef CONFIG_BOOT_MSG_STRING
```